omod/src/main/webapp/resources/scripts/visit/constants.js

```diff
@@ -276,12 +276,15 @@ angular.module('constants', [])
         normalExamFinding: {
             uuid: "3cd750a0-26fe-102b-80cb-0017a47871b2"
         },
-        labTestOrdered: {
+        medicationOrders: {  // 1282
```

Comment:
Why the change here @brandones? Was this a typo on our part you found?

Reply:
I think it had just been named the wrong thing by accident: https://concepts.pih-emr.org/openmrs/dictionary/concept.htm?conceptId=1282

Reply:
Thanks.  Sounds like a good find.